lib/markd/README.md

```diff
@@ -73,7 +73,7 @@ html = renderer.render(document)
 
 ## Performance
 
-First of all, Markd is slower than [Crystal Built-in Markdown](https://crystal-lang.org/api/0.23.0/Markdown.html) which it is a lite version, only apply for generte Cystal documents ([#4496](https://github.com/crystal-lang/crystal/pull/4496), [#4613](https://github.com/crystal-lang/crystal/issues/4613)).
+First of all, Markd is slower than [Crystal Built-in Markdown](https://crystal-lang.org/api/0.23.0/Markdown.html) which it is a lite version, only apply for generate Cystal documents ([#4496](https://github.com/crystal-lang/crystal/pull/4496), [#4613](https://github.com/crystal-lang/crystal/issues/4613)).
```

Comment:
```suggestion
First of all, Markd is slower than [Crystal's Built-in Markdown](https://crystal-lang.org/api/0.23.0/Markdown.html) which is a lite version, only applies for generating Crystal documents ([#4496](https://github.com/crystal-lang/crystal/pull/4496), [#4613](https://github.com/crystal-lang/crystal/issues/4613)).
```

Reply:
FWIW anything in `lib/markd` needs to be merged upstream in https://github.com/icyleaf/markd as it's a lib that checked into VC.

Also related: https://forum.crystal-lang.org/t/markd-donate-or-maintainers-wanted/4500.

Reply:
Does that include this readme? (I'll drop everything applicable.)

Reply:
Probably yea, given it's part of the lib.